documentation/docusaurus.config.js

```diff
@@ -34,15 +34,16 @@ const title = rawProjectName
   })
   .join(' ');
 
+const baseUrl = process.env.PROJECT_NAME || "docs-dev-mode";
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   /*TODO: Change to your project's title and tagline*/
   title: title,
   tagline: 'Owls are cool',
   /*Unless you move this website to a seperate repo don't change url and baseurl.*/
-  url: 'https://'+process.env.ORG_NAME+'.github.io/',
-  baseUrl: '/'+process.env.PROJECT_NAME+'/',
+  url: 'https://'+baseUrl+'.github.io/',
```

Comment:
This does need to be `ORG_NAME` for the GitHub Actions deployment to work universally for GitHub Pages. 
```suggestion
  url: 'https://'+process.env.ORG_NAME+'.github.io/',
```